### PR TITLE
WT3: rewrite for OS4

### PR DESCRIPTION
**Pull request by pb82**
1. fix links in WT3
2. rewrite for OS4 (login, project names, etc.)

### CODE REVIEW

**Comment by tiffanynolan** [comment]
lgtm

File: walkthroughs/3-low-code-api-development/walkthrough.adoc [hunk 1]
```diff
@@ -347,19 +347,17 @@ By default, Fuse Online does not expose an *Integration* to the public internet
 === Find the Greeting Service Internal Hostname
 
 . Login to the link:{openshift-console-url}[OpenShift Console, window="_blank"].
-. Select the project named with the following format: `fuse-<UNIQUE_ID>`.
-. Choose *Applications > Services* from the left menu.
-. Select *i-greeting-integration-{user-username}* from the *Services* list.
-. Note the *Hostname* listed on the *Service Details* screen. It will have a format similar to `i-greeting-integration-{user-username}.fuse-<UNIQUE_ID>.svc.cluster.local`
+. Select the project named with the following format: `redhat-rhmi-fuse`.
```
Comment (evanshortiss) [inline walkthroughs/3-low-code-api-development/walkthrough.adoc:350]: Users will use the shared fuse in OS4?

File: walkthroughs/3-low-code-api-development/walkthrough.adoc [hunk 2]
```diff
@@ -347,19 +347,17 @@ By default, Fuse Online does not expose an *Integration* to the public internet
...
+. Choose *Network > Services* from the left menu.
+. Make sure there is a *i-greeting-integration-{user-username}* item in the *Services* list.
+. Take a note of the hostname: `i-greeting-integration-{user-username}.redhat-rhmi-fuse.svc.cluster.local`.
```
Comment (evanshortiss) [inline walkthroughs/3-low-code-api-development/walkthrough.adoc:353]: QQ. Can the `svc.cluster.local` change? I feel as though I've also seen another variation of that now that I think of it.

File: walkthroughs/3-low-code-api-development/walkthrough.adoc [hunk 3]
```diff
@@ -347,19 +347,17 @@ By default, Fuse Online does not expose an *Integration* to the public internet
...
 
-. Login to OpenShift by using the following command, and entering your password when prompted:
-+
-[subs="attributes+"]
-----
-oc login -u {user-username} {openshift-host}
-----
+. Login to the link:{openshift-console-url}[OpenShift Console, window="_blank"].
+. In the top right dropdown, select *{user-username} > Copy Login Command*.
+. Login with your credentials and click on *Display Token*.
```
Comment (evanshortiss) [inline walkthroughs/3-low-code-api-development/walkthrough.adoc]: "Login with your credentials if/when requested" maybe?

**Review by evanshortiss — CHANGES_REQUESTED** [review]
Minor questions, but LGTM

File: walkthroughs/3-low-code-api-development/walkthrough.adoc (hunk 2 above)
Comment (pb82) [reply walkthroughs/3-low-code-api-development/walkthrough.adoc:353]: I think that value is for services inside a cluster. OS4 does no longer display the hostname in the service details, so I construct the URL directly.

File: walkthroughs/3-low-code-api-development/walkthrough.adoc (hunk 1 above)
Comment (pb82) [reply walkthroughs/3-low-code-api-development/walkthrough.adoc:350]: Good question. No fuse instance got provisioned in the WT namespace, is that expected?

File: walkthroughs/3-low-code-api-development/walkthrough.adoc (hunk 3 above)
Comment (pb82) [reply walkthroughs/3-low-code-api-development/walkthrough.adoc]: Updated

File: walkthroughs/3-low-code-api-development/walkthrough.adoc (hunk 2 above)
Comment (evanshortiss) [reply walkthroughs/3-low-code-api-development/walkthrough.adoc:353]: Gotcha, I think that's what I'm remembering - the 3 vs 4 difference 👍

**Review by evanshortiss — APPROVED** [review]

**Pull request merged**